src/customRules.ts

```diff
@@ -2,5 +2,6 @@ module.exports = {
   'rulesDirectory': ['./customRules'],
   'rules': {
     'trailing-comma-interface': true,
+    'structured-imports': true,
```

Comment:
I'd rather be more specific about the intent of the rule.  Something like `import-type-order`.